src/sardana/pool/poolcontrollers/DummyMotorController.py

```diff
@@ -569,8 +588,9 @@ def PreStartOne(self, axis, pos):
     def StartOne(self, axis, pos):
         #raise Exception("Cannot move on StartOne")
         idx = axis - 1
+        m = self.m[idx]
```

Comment:
I don't know why this is there ... I probably took an old version not cleaned up, I didn't put. 
Yes, it should not be there. Thanks.